packages/dev-tools/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "noImplicitAny": false,
     "outDir": "build/server",
     "rootDir": "server",
-    "allowJs": true,
+    "allowJs": false,
```

Comment:
Would this prevent this from getting copied over? https://github.com/expo/expo-cli/blob/master/packages/dev-tools/server/babel.config.js not sure if it's needed or not though.

Reply:
Hmm, not sure either. I have flipped this flag to get rid of error below:
```
@expo/dev-tools: $ ts-node ./server/extract-fragment-types
@expo/dev-tools: /Users/simek/GitHub/expo-cli/node_modules/ts-node/src/index.ts:755
@expo/dev-tools:           throw new TypeError(`${relative(cwd, fileName)}: Emit skipped`)
@expo/dev-tools:                 ^
@expo/dev-tools: TypeError: ../config/build/index.js: Emit skipped
@expo/dev-tools:     at getOutput (/Users/simek/GitHub/expo-cli/node_modules/ts-node/src/index.ts:755:17)
@expo/dev-tools:     at Object.compile (/Users/simek/GitHub/expo-cli/node_modules/ts-node/src/index.ts:968:32)
```
The problem has been described earlier in `ts-node` repo, and occurs probably due to some config path issues. I will investigate this problem further, there should be better way to fix this than flipping the flag.